Helper/Requests.php

```diff
@@ -341,6 +341,19 @@ public function buildThreeDS2Data($additionalData, $storeId, $request = [])
         return $request;
     }
 
+    /**
+     * @param array $request
+     * @return array
+     */
+    public function buildRedirectData($request = [])
+    {
+        $request['redirectFromIssuerMethod'] = 'GET';
```

Comment:
can you walk us through how this works in the new flow?

Reply:
If the plugin sends `redirectFromIssuerMethod` and `redirectToIssuerMethod` Adyen won't redirect back to the store after 3DS1 auth, but rather it will go to an internal link (termUrl) that contains the data. Then that link will go back to the store via GET and not POST.